mne_bids_pipeline/_config_utils.py

```diff
@@ -423,22 +423,21 @@ def get_mf_ctc_fname(
 def _restrict_analyze_channels(
     inst: RawEpochsEvokedT, cfg: SimpleNamespace
 ) -> RawEpochsEvokedT:
-    if cfg.analyze_channels:
```

Comment:
This should always be true with valid configuration options

Reply:
True but to save us some pain you should add a `Len` or similar annotation to this to make sure it is non-empty

https://github.com/mne-tools/mne-bids-pipeline/blob/b52742e0b1a08deac963f59dafff4ad2d0578a62/mne_bids_pipeline/_config.py#L385

Reply:
(then `pydantic` will throw an error if it's an empty list)

Reply:
Pushed `Annotated[..., MinLen(1)]`